deploy.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash
 
-PROFILE="stell"
+# The following variables must be personalized before script execution
+PROFILE="stell-gold"
```

Comment:
makes sense. i will replace this with a message stating the personalized vars must be exported if not defined.